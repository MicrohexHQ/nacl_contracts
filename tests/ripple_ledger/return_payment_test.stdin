rpc new_ledger s("{\"fee_base\":10,\"fee_ref\":10,\"ledger_hash\":\"D01692280FA2EC56A065AD12373FC782A030AE0BC9DB3C51058A6C3334E76CCA\",\"ledger_index\":6902147,\"ledger_time\":454697770,\"reserve_base\":20000000,\"reserve_inc\":5000000,\"txn_count\":3,\"type\":\"ledgerClosed\",\"validated_ledgers\":\"32570-6902147\"}") *

rpc new_transaction s("{\"Account\":\"rG4dR2LkG9AZRtefoW9hSL8b3XBZgjNowy\",\"Amount\":\"1000000\",\"Destination\":\"rPbFo6KLNSk86hkdDS4NeioJbWxBHvppk\",\"Fee\":\"12\",\"Flags\":0,\"Sequence\":35,\"SigningPubKey\":\"02B3B49241A02B46F2FA047C58F81FD77614F41C526BAF97D0CCCB909A1CA22620\",\"TransactionType\":\"Payment\",\"TxnSignature\":\"3046022100E7CA828DB2EB39E311BAF1D9B743B07CADDEA5492BFC35F673623E41AA895940022100AAEDA8A3782A1A6E05A7DE14483CB3D040C20288005E942E76937EA62860427B\",\"date\":454541660,\"hash\":\"545B1EE473CBBD0C65AD1C590892524BCF7A17D30CD008189D14F33DC6B5441D\",\"inLedger\":6868746,\"ledger_index\":6868746}") *