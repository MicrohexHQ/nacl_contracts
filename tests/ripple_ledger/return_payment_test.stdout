rpc call initiated new_ledger:s:
input 0:  s("{\"fee_base\":10,\"fee_ref\":10,\"ledger_hash\":\"D01692280FA2EC56A065AD12373FC782A030AE0BC9DB3C51058A6C3334E76CCA\",\"ledger_index\":6902147,\"ledger_time\":454697770,\"reserve_base\":20000000,\"reserve_inc\":5000000,\"txn_count\":3,\"type\":\"ledgerClosed\",\"validated_ledgers\":\"32570-6902147\"}")
Got ripple ledger service descriptor 10
Got ripple ledger connection 11
Let's get some account transactions!
rpc call complete new_ledger:s:

rpc call initiated new_ledger:s:
input 0:  s("{\"fee_base\":10,\"fee_ref\":10,\"ledger_hash\":\"D01692280FA2EC56A065AD12373FC782A030AE0BC9DB3C51058A6C3334E76CCA\",\"ledger_index\":6902147,\"ledger_time\":454697770,\"reserve_base\":20000000,\"reserve_inc\":5000000,\"txn_count\":3,\"type\":\"ledgerClosed\",\"validated_ledgers\":\"32570-6902147\"}")
rpc call complete new_transaction:s:

